jobs/correction-ben-statement/data.py

```diff
@@ -0,0 +1,16 @@
+# Populate this list with the existing BEN business identifiers from specified environment
+# This works as a data file for Jupyter notebook used to add Registrar's Notation
+ben_businesses = [
+    "BC0871277",
+    "BC0871062",
+    "BC0871261",
+    "BC0871183",
+    "BC0871181",
+    "BC0871152",
+    "BC0871062",
+    "BC0871139",
+    "BC0871073",
+    "BC0870819",
+    "BC0870818"
```

Comment:
Is this the list of businesses in Prod?

If not, when do we update and commit this file?

Reply:
Since this is a committed file, how can we test this in Test environment?

Reply:
This is not a list from PROD. I think we will have to get the list from PROD around/closer to the deployment date, dump the list here and then run script.
Yes. I think we can test this in TEST once released to test. We just need to get the test data (identifiers).
@vysakh-menon-aot please add if I am missing anything

Reply:
Consider this as a one time script, only difference is its a jupiter notebook. We are not deploying this job in openshift. We have to manually execute this in each environment with the list of identifier.

Reply:
yes. I will add all the details in the ticket

Reply:
@ketaki-deodhar its better not to keep all these identifiers in repo. Keep 1 or 2 as a sample and remove rest of it